content/postgresql/concepts/subscripts/subscripts.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: 'Subscripts'
+Description: 'In PostgreSQL, subscripts are used to perform various operations on arrays. They offer a concise way to handle and manage array data.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Arrays'
+  - 'Database'
+  - 'Data Structures'
+  - 'PostgreSQL'
+  - 'SQL'
+CatalogContent:
+  - 'paths/analyze-data-with-sql'
+  - 'paths/data-science'
+---
+
+**Subscripts** are a fundamental concept in PostgreSQL, used to access and manipulate elements within an array. By using subscripts, it becomes easy to retrieve, update, or remove elements from an array by referring to their specific positions.
+
+## Syntax
+
+For a one-dimensional array, the following syntax is used:
+
+```pseudo
+array_name[subscript]
+```
+- `array_name`: The name of the array being accessed.
+- `subscript`: The index position of the element to be accessed within the array.
+
+Alternatively, for multi-dimensional array the following syntax is used:
+
+```pseudo
+array_name[subscript_1, subscript_2, ..., subscript_n]
```

Comment:
In PostgreSQL, multi-dimensional array elements are accessed using nested square brackets, where each set of brackets corresponds to a specific dimension. Refer [docs](https://www.postgresql.org/docs/current/arrays.html) for more info.

The syntax for accessing multi-dimensional arrays in Postgres is wrong, and the correct syntax is as follows -

```suggestion
```pseudo
array_name[subscript_1][subscript_2]...[subscript_n]
```